bytes/src/main/java/net/consensys/cava/bytes/Bytes.java

```diff
@@ -314,6 +314,79 @@ static Bytes ofUnsignedInt(long value) {
     return Bytes.wrap(res);
   }
 
+  /**
+   * Return a 8 bytes value corresponding to the provided value interpreted as an unsigned long value.
+   *
+   * @param value The value, which must fit an unsigned long.
+   * @return A 8 bytes value corresponding to {@code v}.
+   * @throws IllegalArgumentException if {@code v < 0} or {@code v} is too big to fit an unsigned 8-bytes int (that is,
+   *         if {@code v >= (1L << 64)}).
+   */
+  static Bytes ofUnsignedLong(long value) {
+    checkArgument(
+        value >= 0 && value <= BytesValues.MAX_UNSIGNED_LONG,
+        "Value %s cannot be represented as an unsigned int (it is negative or too big)",
+        value);
+    byte[] res = new byte[8];
+    res[0] = (byte) ((value >> 56) & 0xFF);
+    res[1] = (byte) ((value >> 48) & 0xFF);
+    res[2] = (byte) ((value >> 40) & 0xFF);
+    res[3] = (byte) ((value >> 32) & 0xFF);
+    res[4] = (byte) ((value >> 24) & 0xFF);
+    res[5] = (byte) ((value >> 16) & 0xFF);
+    res[6] = (byte) ((value >> 8) & 0xFF);
+    res[7] = (byte) (value & 0xFF);
+    return Bytes.wrap(res);
+  }
+
+  /**
+   * Return a 4 bytes little-endian encoded value corresponding to the provided value interpreted as an unsigned int
+   * value.
+   *
+   * @param value The value, which must fit an unsigned int.
+   * @return A 4 bytes value corresponding to {@code v}.
+   * @throws IllegalArgumentException if {@code v < 0} or {@code v} is too big to fit an unsigned 4-bytes int (that is,
+   *         if {@code v >= (1L << 32)}).
+   */
+  static Bytes ofUnsignedIntAsLittleEndian(long value) {
```

Comment:
`ofLittleEndianUnsignedInt`

Reply:
Or use a big/little param

Reply:
maybe the param, as an overloaded method. big endian clearly should be default.